CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ accidentally triggering the load of a previous DB version.**
 * #5317 Fix some incorrect memory handling
 * #5367 Rename columns in old-style continuous aggregates
 * #5384 Fix Hierarchical Continuous Aggregates chunk_interval_size
+* #5304 Support sub-second intervals in hierarchical caggs
```

Comment:
I guess this issue is a bugfix so we should mention it here

Reply:
should I make any changes here then?

Reply:
```suggestion
* #5304 Fix sub-second intervals in Hierarchical Continuous Aggregates
```